src/styles/index.scss

```diff
@@ -1,3 +1,4 @@
-body {
-  margin: 0;
-}
+@import 'reset';
+@import 'card';
+@import 'stars';
```

Comment:
According to the code style rules, the 'variables' file should be imported before any other files to ensure that variables are available for use in other stylesheets. Consider moving the import statement for 'variables' to the top.